tests/src/test/scala/cats/tests/NonEmptyChainSuite.scala

```diff
@@ -60,118 +62,118 @@ class NonEmptyChainSuite extends NonEmptyCollectionSuite[Chain, NonEmptyChain, N
   }
 
   test("show") {
-    Show[NonEmptyChain[Int]].show(NonEmptyChain(1, 2, 3)) should ===("NonEmptyChain(1, 2, 3)")
+    assert(Show[NonEmptyChain[Int]].show(NonEmptyChain(1, 2, 3)) === ("NonEmptyChain(1, 2, 3)"))
   }
 
   test("size is consistent with toChain.size") {
     forAll { (ci: NonEmptyChain[Int]) =>
-      ci.size should ===(ci.toChain.size)
+      assert(ci.size === (ci.toChain.size))
     }
   }
 
   test("filterNot and then exists should always be false") {
     forAll { (ci: NonEmptyChain[Int], f: Int => Boolean) =>
-      ci.filterNot(f).exists(f) should ===(false)
+      assert(ci.filterNot(f).exists(f) === (false))
     }
   }
 
   test("filter and then forall should always be true") {
     forAll { (ci: NonEmptyChain[Int], f: Int => Boolean) =>
-      ci.filter(f).forall(f) should ===(true)
+      assert(ci.filter(f).forall(f) === (true))
```

Comment:
an alternative could be
```suggestion
      assert(ci.filter(f).forall(f))
```